airbyte-integrations/connectors/source-trello/source_trello/spec.json

```diff
@@ -25,6 +25,12 @@
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}Z$",
         "description": "UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated.",
         "examples": ["2021-03-01T00:00:00.000Z"]
+      },
+      "board_ids": {
+        "type": "string",
```

Comment:
could you make this an array whose items are of type string? it will render in a more friendly way on the UI (and is probably the better way to represent this)

Reply:
Thanks for the suggestion @sherifnada I will change accordingly.

Reply:
![Screenshot 2021-11-24 at 12 07 19](https://user-images.githubusercontent.com/176678/143227375-7fb0dd9e-5de4-485f-8cbd-2de86fc5c93e.png)